src/features/user/MFV2/ProfileCard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ProfileProps } from '../../../common/types/profile';
+import { Avatar } from '@mui/material';
+import getImageUrl from '../../../../utils/getImageURL';
+import styles from './ProfileCard.module.scss';
+import {
+  DefaultUserProfileImage,
+  SettingsIcon,
+} from '../../../../../public/assets/images/icons/ProfilePageV2Icons';
+import { useLocale, useTranslations } from 'next-intl';
+import ProfileActions from './ProfileCardButton/ProfileActions';
+import Link from 'next/link';
+
+const ProfileCard = ({ userProfile, authenticatedType }: ProfileProps) => {
```

Comment:
1. I'd suggest setting up interfaces for this component in the same file, as we don't need it elsewhere, and using the interfaces defined for v1 can lead to unintended code linkages.
2. `authenticatedType` - perhaps `profilePageType` or simply `type` conveys the purpose better. I could be logged in and be viewing my public profile.

Reply:
This interface is being used at multiple places for the new version, to prevent linkage to old code I've created a new interface